src/EnergyPlus/DataZoneEquipment.hh

```diff
@@ -614,7 +638,7 @@ struct DataZoneEquipmentData : BaseGlobalStruct
     int NumOfZoneEquipLists = 0;
     Array1D<Avail::Status> ZoneEquipAvail;
     Array1D<DataZoneEquipment::EquipConfiguration> ZoneEquipConfig;
-    EPVector<DataZoneEquipment::EquipConfiguration> spaceEquipConfig;
+    Array1D<DataZoneEquipment::EquipConfiguration> spaceEquipConfig;
```

Comment:
Make this the same type as `ZoneEquipConfig` so they can be passes as a parameter to a common function.